versioned/persist/adapter/src/main/java/org/projectnessie/versioned/persist/adapter/spi/TracingDatabaseAdapter.java

```diff
@@ -102,6 +104,13 @@ public Map<Key, ContentAndState> values(
     }
   }
 
+  @Override
+  public Stream<CommitLogEntry> fetchCommitLogEntries(Stream<Hash> hashes) {
+    try (Traced ignore = trace("fetchCommitLogEntries")) {
```

Comment:
+1 to consistency. However, I'm not sure our overall approach to tracing stream processing is the best we could have under the OpenTracing framework. Substantial work can be performed by inner code during stream operations, but currently it will be attributed to the outer span, AFAIK. This is beyond the scope of this PR, of course. I would like to revisit this at some point, though.